src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -515,6 +515,55 @@ public string[] ParameterSetName
         private string[] _parameterSetName;
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 5, but got 0.214421204196174.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1961
1961:             $pathologicalRatio | Should -BeGreaterThan 5
```